src/DatabaseConnection.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
```

Comment:
This file is intended for using database connection in several files, for now is not completed and is not in use yet.
You can skip it.